src/plot_styles.py

```diff
@@ -124,6 +133,8 @@ class PlotStylesWindow(Adw.Window):
     minor_tick_width = Gtk.Template.Child()
     major_tick_length = Gtk.Template.Child()
     minor_tick_length = Gtk.Template.Child()
+    set_active_style = Gtk.Template.Child()
+    set_active_style_row = Gtk.Template.Child()
```

Comment:
Is there a reason to name this row and keep a reference? It is not accessed in code, just the switch.